src/Classes/MyRadio/CoreUtils.php

```diff
@@ -553,6 +553,22 @@ public static function getAllPermissions()
         );
     }
 
+    private static function comparePermission($perm1, $perm2)
+    {
+        if ($perm1['value'] === $perm2['value']) {
+            return 0;
+        } elseif ($perm1['value'] < $perm2['value']) {
+            return -1;
+        } else {
+            return 1;
+        }
+    }
+
+    public static function diffPermissions($perms)
```

Comment:
maybe "getMissingPermissions" instead? diffPermissions doesn't really tell you what the function does, imo